deps/v8/src/base/platform/platform.h

```diff
@@ -35,7 +35,6 @@ namespace std {
 int signbit(double x);
 }
 # endif
-#include <alloca.h>
```

Comment:
This had been [added by floating a patch](3589a62104eac1daae782ab479bec09f4df4dc9a) that fixes build issues on SmartOS. In the meantime, I believe most of the changes from this patch except the inclusion of the `alloca.h` header on Solaris derivatives have been upstreamed.

Floating this patch again would solve this build issue on SmartOS: http://jenkins.nodejs.org/job/node-test-commit-unix/DESTCPU=ia32,label=smartos/13/console.

We should however probably upstream it too.